tests/conftest.py

```diff
@@ -420,7 +420,7 @@ def _discover_environment(config):
             logger.warning('Cannot get git repository information.')
     try:
         env['has_slurm'] = shutil.which('sbatch') is not None
-        if 'has_slurm' not in env:
+        if 'has_slurm' not in env or not env['has_slurm']:
```

Comment:
Wouldn't `if not env.get('has_slurm')` be simpler?  But either way, fix looks correct to me!

Reply:
Yes, yes it would, of course, exactly because we know that the entry is there